pkg/substitution/substitution_test.go

```diff
@@ -174,7 +174,7 @@ func TestValidateVariablePs(t *testing.T) {
 			vars:   sets.NewString("foo.bar.baz"),
 		},
 		expectedError: &apis.FieldError{
-			Message: `Invalid referencing of parameters in --flag=$(params.foo.bar.baz) !!! You can only use the dots inside single or double quotes. eg. $(params["org.foo.blah"]) or $(params['org.foo.blah']) are valid references but NOT $params.org.foo.blah.`,
+			Message: `extractVariablesFromString failed: invalid referencing of parameters in --flag=$(params.foo.bar.baz) !!! You can only use the dots inside single or double quotes. eg. $(params["org.foo.blah"]) or $(params['org.foo.blah']) are valid references but NOT $(params.org.foo.blah)`,
```

Comment:
I don't think we want the message returned to users to contain the name of the function that failed; it should just explain what is invalid and why.

Reply:
changed. Thanks!